### PR TITLE
Update core dependencies

### PR DESCRIPTION
**Pull request by ChrisSchinnerl**
Updating the core dependencies on `master` to make sure we get all the fixes into `v1.1.0`.

### CODE REVIEW

**Review by peterjan — APPROVED** [review]
LGTM

**Comment by n8maninger** [comment]
@ChrisSchinnerl removing the prefixes broke the hosts table. RHP2 settings stored in the database have the `addr:` prefix so can't be serialized. I had to reset the scans on our databases manually.

**Review by n8maninger — CHANGES_REQUESTED** [review]
requesting changes per my comment -^

File: stores/sql/sqlite/migrations/main/migration_00019_scan_reset.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+UPDATE hosts SET settings = NULL, price_table = NULL, last_scan = NULL, last_scan_success = NULL, second_to_last_scan_success = NULL WHERE 1=1;
```
Comment (n8maninger) [inline stores/sql/sqlite/migrations/main/migration_00019_scan_reset.sql:1]: Is settings = NULL okay? The default `INSERT hosts` looks like it does an empty table.

Comment (ChrisSchinnerl) [reply stores/sql/sqlite/migrations/main/migration_00019_scan_reset.sql]: Good question. Haven't had time to test it yet

**Review by n8maninger — DISMISSED** [review]

**Review by peterjan — DISMISSED** [review]

**Comment by ChrisSchinnerl** [comment]
Tested this more times than I'd like to admit. 5+ times on testnet and also icarus. 
The changes to the contractor were necessary to avoid migration due to the unscanned hosts.

**Review by n8maninger — APPROVED** [review]

**Pull request merged**